### ISSUE TITLE
AbstractMethodError, An API incompatibility was  encountered while executing on checkstyle 8.0

### ISSUE THREAD

**Issue by romani** [sevntu-checkstyle/sevntu.checkstyle#597]
when sevntu 1.24.0 is used with checkstyle 8.0, build results in
```
[ERROR] Failed to execute goal org.apache.maven.plugins:maven-checkstyle-plugin:2.17:check
 (sevntu-checkstyle-check) on project checkstyle: Execution sevntu-checkstyle-check
 of goal org.apache.maven.plugins:maven-checkstyle-plugin:2.17:check failed: 
An API incompatibility was encountered while executing 
org.apache.maven.plugins:maven-checkstyle-plugin:2.17:check: 
java.lang.AbstractMethodError: null
```

**Comment by romani** [comment]
fix is already merged at #596, we just need to do release of it.

**Comment by romani** [comment]
fix is released in 1.24.1
